src/rewrite.rs

```diff
@@ -39,4 +40,18 @@ impl<'a> RewriteContext<'a> {
             block_indent: self.block_indent + self.config.tab_spaces,
         }
     }
+
+    pub fn snippet(&self, span: Span) -> Option<String> {
```

Comment:
Should we unwrap it in here? Providing a bad span is always a programming error, correct?